lib/opensusebasetest.pm

```diff
@@ -388,8 +387,7 @@ sub export_logs_desktop {
     }
     $log_path = '/home/bernhard/.local/share/xorg';
```

Comment:
I would go as far and use /home/*/.local/share/xorg so that we don't need to care about the user, but looks good